client/src/Components/PreNavBar.js

```diff
@@ -3,15 +3,23 @@ import { Link } from "react-router-dom";
 import AppBar from '@material-ui/core/AppBar';
 import Toolbar from '@material-ui/core/Toolbar';
 import Button from '@material-ui/core/Button';
+import Logout from './LogNavBar'
```

Comment:
This should not be imported here, switching between navs is handled at the top level.